MoviesStoreApp/MoviesStoreApp/Controllers/MovieInfoViewController.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  MovieInfoViewController.swift
+//  MoviesStoreApp
+//
+//  Created by Александр Андреевич Щепелин on 26.10.2022.
+//
+
+import SafariServices
+import UIKit
+
+/// Экран с описанием фильма
+final class MovieInfoViewController: UIViewController {
+    // MARK: - Private Visual Components
+
+    private let movieInfoTableView = UITableView()
+
+    // MARK: - Public Properties
+
+    var movieInfo: MovieInfo?
+    var movieID: Int?
+
+    // MARK: - LifeCycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+    }
+
+    // MARK: - Private Methods
+
+    private func setupUI() {
+        setupTableView()
+        movieInfoTableView.delegate = self
+        movieInfoTableView.dataSource = self
+        moviesCollection()
+        setupConstraints()
+    }
+
+    private func moviesCollection() {
+        guard let url = URL(string: "\(URLRequest.baseURL + "\(movieID ?? 0)" + URLRequest.apiKey)")
```

Comment:
```suggestion
        guard let url = URL(string: "")
```
интерполяция или URLComponents